### DISCUSSION TITLE
It does not recognize the new version of BIOS

### DISCUSSION THREAD

**Discussion by ikspekt** [seerge/g-helper#1085] [Q&A]
![스크린샷 2023-08-15 023719](https://github.com/seerge/g-helper/assets/136056477/a24daa3f-51b9-4492-a856-d6f9ef2cabcc)

I updated the bios to version 315, but it doesn't recognize it. Also I cannot download a new version. Is there a delay?

![스크린샷 2023-08-15 023952](https://github.com/seerge/g-helper/assets/136056477/6877b336-b1b4-4a8f-983d-0e5474904b95)
+) Ghelper cannot stop some asus programs since I updated bios.

My laptop is G15 GA503RM.
Thank you for providing a great program :)

**Comment by seerge** [comment]
G-Helper uses same feed as MyAsus does (for page where you can see all drivers / bioses)

And there new bios is still not published by some reason (i have similar situation for my G14). 

I guess it will appear there in a week or two 

MyAsus also fetches updates on its main screen somehow, but i don't know feed for that (as i have already updated and i see nothing there)
